chicago-fullnote-bibliography.csl

```diff
@@ -758,7 +758,19 @@
     <choose>
       <if variable="issued">
         <choose>
-          <if type="legal_case">
+          <if type="article-journal" match="any">
+            <group delimiter=" ">
```

Comment:
no need for the group here since there's only one element (dates have spacing of their own)

Reply:
Removed the group.